static-analysis-plugin/plugin/src/test/groovy/com/novoda/test/PenaltyExtensionSubject.groovy

```diff
@@ -23,20 +22,25 @@ class PenaltyExtensionSubject extends Subject<PenaltyExtensionSubject, PenaltyEx
         }
     }
 
-    private PenaltyExtensionSubject(FailureStrategy failureStrategy, @Nullable PenaltyExtension actual) {
-        super(failureStrategy, actual)
-    }
-
     public static PenaltyExtensionSubject assertThat(PenaltyExtension extension) {
         assertAbout(FACTORY).that(extension)
     }
 
+
+    public static PenaltyExtensionSubject assumeThat(PenaltyExtension extension) {
```

Comment:
I'm a bit confused... these new public methods aren't being used, is that gonna be part of the next PR?